src/lua/popen.c

```diff
@@ -1250,8 +1203,6 @@ luaT_popen_parse_mode(struct lua_State *L, int idx)
  *                getpgrp() fails in the parent process.
  * - SystemError: (temporary restriction) the parent process
  *                has closed stdin, stdout or stderr.
- * - OutOfMemory: unable to allocate the handle or a temporary
- *                buffer.
```

Comment:
We should also update the documentation: https://www.tarantool.io/en/doc/latest/reference/reference_lua/popen/#popen-new

The same for `:read()`.

Reply:
Right. Added a doc bot request to the relevant commit.